### ISSUE TITLE
RCS1213 triggered when member is passed from primary constructor to base constructor

### ISSUE THREAD

**Issue by gtbuchanan** [dotnet/roslynator#1342]
**Product and Version Used**: 
Roslynator 4.7.0

**Steps to Reproduce**:
1. Create a class with a primary constructor that inherits from another class with a constructor that has arguments.
2. Call the base constructor from the primary constructor, passing a member of the class as a parameter.

**Actual Behavior**:
```csharp
using System;

internal abstract class BaseClass(Action action)
{
    public Action Action { get; } = action;
}

internal sealed class ChildClass() : BaseClass(DoAction)
{
    private static void DoAction() // BUG: RCS1213 triggered here
    {
    }
}
```

**Expected Behavior**:
[RCS1213](https://josefpihrt.github.io/docs/roslynator/analyzers/RCS1213/) should not be triggered.
